### ISSUE TITLE
Handlebars Partials crash kss-node

### ISSUE THREAD

**Issue by dotherightthing** [kss-node/kss-node#205]
I am using `.hbs` files with success.

    Markup: ../../modules/my-module.hbs

However I wish to reuse `.hbs` files with different data, to show different variations in the styleguide (`{{modifier_class}}` is insufficient to show markup variations).

`kss-node` automatically loads the JSON data with the same name as the `.hbs` markup file (`my-module.hbs` uses `./my-module.json`)

Therefore I am creating a separate `.hbs` file for each variation, to match the variation data files (`my-module_variation.hbs` uses `./my-module_variation.json`)

Because I don't want or need a different `.hbs` file for each variation, only a different `.json` file, I am trying to achieve this by using partials, so that the `.hbs` variations just load the 'default' `.hbs` file:

    // my-module_variation1.hbs
    {{> my-module}}

    // my-module_variation2.hbs
    {{> my-module}}

However when I do this kss-node crashes with no useful error message.

> Running "kss:dist" (kss) task
>> kss-node '../../resources/build/styles/' '../../resources/build/docs/styleguide/generated' --template ../../node_modules/ct__kss-node-template --config ../../.kss-project-config.json
>> Error: Command failed: /bin/sh -c node /Users/Dan/Websites/outwardbound-2015/wp-content/themes/outwardbound2014/node_modules/ct__kss-node-template/node_modules/grunt-kss/node_modules/kss/bin/kss-node '../../resources/build/styles/' '../../resources/build/docs/styleguide/generated' --template ../../node_modules/ct__kss-node-template --config ../../.kss-project-config.json
>> Code:

Is my syntax wrong or is loading of partials not supported?

Thanks.

**Comment by JohnAlbin** [comment]
Not sure. Maybe you need quotes around the names?

https://github.com/kss-node/kss-node/blob/master/demo/forms/forms.hbs uses Handlebars partials.

**Comment by dotherightthing** [comment]
@JohnAlbin Thanks, it looks like I don't fully understand how Handlebars works, as I've only been using it for kss-node, prior to that I used Mustache.

I couldn't get `{{> "index.of.mymodule"}}` working in my code so I need to play around with your demo to see how my code differs from yours.

However I did get `{{> "my-module_variation1.hbs" value1="foo" value2="bar"}}` working so I'm a couple of steps closer to DRY documentation.

Cheers,
Dan
